src/frontend/Player/createPlyrPlayer.ts

```diff
@@ -232,39 +225,39 @@ export const createPlyrPlayer = (
   player.on('loadedmetadata', initialize);
   player.on('loadeddata', initialize);
 
-  player.on('playing', (event) => {
+  player.on('playing', () => {
     xapiStatement.played({
       time: player.currentTime,
     });
 
     changePlayButtonAriaLabel(player.config.i18n.pause);
   });
 
-  player.on('pause', (event) => {
+  player.on('pause', () => {
     xapiStatement.paused({
       time: player.currentTime,
     });
 
     changePlayButtonAriaLabel(player.config.i18n.play);
   });
 
-  /**************** Seeked statement ***********************/
-  player.on('timeupdate', (event) => {
-    if (true === isInitialized && false === player.seeking) {
-      currentTime = player.currentTime;
+  /** ************** Seeked statement ********************** */
+  player.on('timeupdate', () => {
+    if (isInitialized === true && player.seeking === false) {
+      ({ currentTime } = player);
```

Comment:
Not sure this is clearer than what was before. Maybe we can turn this option off?